### PR TITLE
Fix some linter issues

### PR DESCRIPTION
**Pull request by guenhter**
This are just the linter issues which could be fixed automated by `ansible-lint --write` but I thought it is already worth creating a PR because this is already quite comprehensive.

### CODE REVIEW

**Comment by riemers** [comment]
As you can see, i turned travis back on, it was changed somehow. I am also looking at the ansible galaxy thing, but boy did they screw that up.  Docs wise i cannot find 1.2.3 on how to upload a new version. Would that not be the first thing you would document or at least mention in your new site ;)

**Comment by guenhter** [comment]
I'll tried to run the tests locally and for Linux they start to fail in a early stage. Should I have a look to fix the local tests?

**Comment by riemers** [comment]
If possible. Should have enough credits on CI to do some tests. Presume some updates where made. Also the role update to galaxy seems broken.

Verzonden vanuit Outlook voor iOS<https://aka.ms/o0ukef>
________________________________
Van: Günther Grill ***@***.***>
Verzonden: Monday, October 9, 2023 7:39:19 AM
Aan: riemers/ansible-gitlab-runner ***@***.***>
CC: Erik-jan Riemers ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [riemers/ansible-gitlab-runner] Fix some linter issues (PR #286)


I'll tried to run the tests locally and for Linux they start to fail in a early stage. Should I have a look to fix the local tests?

**Comment by riemers** [comment]
Also don't forget to slap your name somewhere since i really appreciate the work.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 # This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
 #
 # You can adjust the behavior by modifying this file.
 # For more information, see:
 # https://github.com/actions/stale
 
-name: 'Close stale issues and PRs'
-on:
+name: Close stale issues and PRs
+true:
```
Comment (gardar) [inline .github/workflows/stale.yml:9]: This is not right and breaks the workflow

Comment (guenhter) [reply .github/workflows/stale.yml:9]: Outch. Mabe we can fix this with one of the next pull requests.

**Pull request merged**